conda-forge.yml

```diff
@@ -1,3 +1,8 @@
+azure:
+  settings_linux:
+    timeoutInMinutes: 1
+  settings_osx:
+    timeoutInMinutes: 1
```

Comment:
Amazing!

shall we revert the pinning changes?

Reply:
I would like to do this in a separate PR. They make take some time due to `nvcc` weirdness and I simply want to minimise the load impact our work on this feedstock has.

Reply:
i meant

```
git checkout main -- .ci_support
```
to undo the changes in the ci_support files that relate to pinning upgrades that didn't happen.

Reply:
Ah, got it!